dialogflow/api/DialogflowSamples/ContextManagement.cs

```diff
@@ -121,7 +127,11 @@ public class DeleteAllOptions : OptionsWithProjectIdAndSessionId { }
         public static int DeleteAllContexts(string projectId, string sessionId)
         {
             var client = ContextsClient.Create();
-            client.DeleteAllContexts(new SessionName(projectId, sessionId));
+            var deleteAllContextsRequests = new DeleteAllContextsRequest
+            {
+                Parent = SessionName.FromProjectSession(projectId, sessionId).ToString()
+            };
+            client.DeleteAllContexts(deleteAllContextsRequests);

```

Comment:
As of Friday, you can pass the session name in directly. (Will need to do another release, of course...)